### ISSUE TITLE
Create WCAG 2.0 AA checklist for eAPD

### ISSUE THREAD

**Issue by nkkl** [18F/cms-hitech-apd#1260]
The WCAG 2.0 AA assessment criteria are not particularly approachable. We have the beginnings of a simplified AA assessment, and we can turn it into a checklist for eAPD.

### This task is done when...
 - [ ] We have a list of only the WCAG 2.0 AA criteria that apply to eAPD
 - [ ] The list can be used repeatedly to test the product

**Comment by nkkl** [comment]
Enables #1259

**Comment by NAretakis** [comment]
Checklist created, needs to be cleaned up and reviewed with devs.

**Comment by jeromeleecms** [comment]
@hbillings has graciously agreed to review the cleaned up checklist. Once it's done, will review and determine next steps (meeting to review or finalize for use).

**Comment by hbillings** [comment]
Linking it here so I don't lose it again :) https://docs.google.com/spreadsheets/d/1g2gZMSH2tzZeNd75Z4Cxi9Y4ZjblxVOL_UhUWVHYOZs/edit#gid=398377983

**Comment by hbillings** [comment]
After experimenting with a few different tools, one of my colleagues reminded me about HTML_CodeSniffer, which has by far the most readable report output I've found: https://squizlabs.github.io/HTML_CodeSniffer/

The other excellent thing about it is that it lists in detail the items it can and cannot detect: http://squizlabs.github.io/HTML_CodeSniffer/Standards/WCAG2/

We should be able to cross-reference that list with the existing spreadsheet to note things that must be manually checked for.

**Comment by jeromeleecms** [comment]
Finished crosswalk - will need to a do a final walk through of the revised checklist before starting #1259.

**Comment by jeromeleecms** [comment]
Had a good review of the crosswalk to discuss the importance of walking through the experience vs. making sure the architecture is correct (and delegating issues that appear) . Will close this issue and start working on #1259 using the applicable testing scenarios as a reference: https://github.com/18F/cms-hitech-apd/wiki/testing-scenarios
